setup.cfg

```diff
@@ -1,6 +1,85 @@
-[bdist_rpm]
-packager = Sylvain Thenault  <sylvain.thenault@logilab.fr>
-provides = pylint
+[metadata]
+name = pylint
+license = GPL-2.0-or-later
+license_file = COPYING
+description = python code static checker
+keywords = static code analysis linter python lint
+long_description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+url = https://github.com/PyCQA/pylint
+project_urls =
+    What's New=https://pylint.pycqa.org/en/latest/whatsnew/
+    Bug tracker=https://github.com/PyCQA/pylint/issues
+    User manual=http://pylint.pycqa.org/en/latest/
+    Contributing=http://pylint.pycqa.org/en/latest/development_guide/contribute.html
+    Technical references=http://pylint.pycqa.org/en/latest/technical_reference/index.html
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License (GPL)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Debuggers
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+
+[options]
+zip_safe = True
+packages = find:
+python_requires = ~=3.6
+py_modules = easy_install
+setup_requires =
```

Comment:
If unused, it can probably be removed.

Reply:
I'm planning on maybe using ``pbr`` or ``setuptools_scm`` or something else (?) to handle the version. Do you use anything like that yourself ? Seems like it could remove a whole lot of manipulation of the version during release too...

Reply:
I haven't used any of them so I can't give you an advice there unfortunately.

Reply:
``version = attr: pylint.__version__`` make a temporary solution. Might change that later but we'd have to change the tag from ``pylint-x.y.z`` to ``x.y.z`` or ``vx.y.z``

Reply:
> I'm planning on maybe using `pbr` or `setuptools_scm` or something else (?) to handle the version. Do you use anything like that yourself ? Seems like it could remove a whole lot of manipulation of the version during release too...

from a packager perspective, pbr and setuptools_scm are pretty annoying :) if you just need to read the version of pylint from withing setup.py you can choose some of the options available here: https://packaging.python.org/guides/single-sourcing-package-version/